### ISSUE TITLE
Mock vs stub

### ISSUE THREAD

**Issue by onmyway133** [Skyscanner/Dixie#6]
In the REAME, you say

> The idea of changing an object's behaviour is not new, it is called mocking

You must mean "stubbing" ?

**Comment by WiesnerPeti** [comment]
Hi @onmyway133 

the sentences indeed would be correct with the word "stubbing". The original intention was to give a small introduction to this type of testing. 

We are about to update the read.me, we may rephrase this to avoid misunderstandings.

Thank you for your contribution!

Cheers,
Peter

**Comment by onmyway133** [comment]
@WiesnerPeti you're welcome :] But how is this over Quick/OCMock/Kiwi, their syntax is easier to use. Can you state your advantages in the README as well
